modules/database/sqlite/db.copy.php

```diff
@@ -19,10 +19,15 @@
 	return;
 }
 
-// Define SQLite constant.
+// Constant for backward compatibility.
 if ( ! defined( 'DATABASE_TYPE' ) ) {
 	define( 'DATABASE_TYPE', 'sqlite' );
 }
 
+// Define SQLite constant.
+if ( ! defined( 'DB_ENGINE' ) ) {
+	define( 'DB_ENGINE', 'sqlite' );
```

Comment:
```suggestion
	define( 'DB_ENGINE', DATABASE_TYPE );
```

Backwards compatibility?

Reply:
Currently, we're using `DATABASE_TYPE` as a constant, so that's what existing users will have in their `db.php` file - and maybe define it in their `wp-config.php` file.
In the refactor, I started looking at backporting  SQLite in Core (https://github.com/WordPress/wordpress-develop/pull/3220) and when doing so, I realized that `DATABASE_TYPE` will not be consistent with all the other constants in Core (like `DB_NAME`, `DB_USER`, `DB_HOST` etc), so I renamed it to `DB_ENGINE` for consistency.
This addition is simply to provide backwards-compatibility with users who were using the previous implementation AND were using a `DATABASE_TYPE` constant in their `wp-config.php` file to control which database gets used 👍